subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DependencyUnresolvedModuleIntegrationTest.groovy

```diff
@@ -233,20 +233,25 @@ class DependencyUnresolvedModuleIntegrationTest extends AbstractHttpDependencyRe
 
         when:
         moduleA.pom."$action"()
+        if (!abort) {
+            backupA.pom."${action}"()
+        }
```

Comment:
I think here we should make the first repo broken/blocking to test the abort mechanism. Making the second one fail doesn't make much sense.

Reply:
Both of the repositories are broken/blocking. In the 'abort' case, the second repository is never called.

Reply:
Yes, sorry for the misunderstanding.